Library/ViewModels/LiveStreamDiscoveryLiveNowCellViewModel.swift

```diff
@@ -48,6 +50,14 @@ LiveStreamDiscoveryLiveNowCellViewModelInputs, LiveStreamDiscoveryLiveNowCellVie
     self.creatorLabelText = liveStreamEvent
       .map { Strings.Creator_name_is_live_now(creator_name: $0.creator.name) }
 
+    self.numberPeopleWatchingHidden = liveStreamEvent.map { $0.numberPeopleWatching == nil }
+
+    self.numberPeopleWatchingText = liveStreamEvent.map {
+      guard let numberWatching = $0.numberPeopleWatching else { return "0" }
+
+      return Format.wholeNumber(numberWatching)
```

Comment:
that's cool too. if we had the `>>>` operator we could go full point free:

```swift
liveStreamEvent
  .map(
    LiveStreamEvent.lens.numberPeopleWatching.view
      >>> coalesceWith(0)
      >>> Format.wholeNumber
)
```

lol